legend-pure-m2-external-json/src/main/resources/platform/json/json.pure

```diff
@@ -81,6 +81,7 @@ Class meta::json::JSONConfig
 
 Class meta::json::JSONDeserializationConfig extends JSONConfig
 {
+    removeType: Boolean[0..1];
```

Comment:
boolean's default is usually false, better call it retainTypeField (true means don't remove type field, [] or false means remove it, which is the default behaviour)

Reply:
[] or default behaviour is actually dont remove types, added this field because I needed to remove type for a particular json deserialization